docusaurus.config.ts

```diff
@@ -176,6 +176,7 @@ const config = (async (): Promise<Config> => {
           { to: '/blog', label: 'Blog', position: 'left' },
           { to: '/community', label: 'Community', position: 'left' },
           { type: 'doc', docId: 'intro', position: 'left', label: 'Docs' },
+/*          { type: 'doc', docId: 'swag', position: 'left', label: 'Get Swag' },*/
```

Comment:
This can be removed

Reply:
Removed comment and added link to footer instead